docs/ldml/tr35-dates.md

```diff
@@ -67,7 +67,7 @@ The LDML specification is divided into the following parts:
       * 4.5.1.2 [Variable periods](#Variable_periods)
       * 4.5.1.3 [Parsing Day Periods](#Parsing_Day_Periods)
 * 5 [Time Zone Names](#Time_Zone_Names)
-  * Table: [<timeZoneNames> Elements Used for Fallback](#timeZoneNames_Elements_Used_for_Fallback)
+  * Table: [&lt;timeZoneNames&gt; Elements Used for Fallback](#timeZoneNames_Elements_Used_for_Fallback)
```

Comment:
again was this a manual change? because this line is generated by code automatically.  if it's a bug in the processor we can deal with that, but otherwise this would be overwritten next time someone manually runs ToC and it also would not have an effect on the final HTML, which always gets a ToC generation automatically.


It might be better just to drop the `<>` from the heading.

Reply:
I will do that, thanks.